Sources/_MatchingEngine/Regex/Parse/Parse.swift

```diff
@@ -273,6 +273,12 @@ extension Parser {
         continue
       }
 
+      // Quoted sequence.
+      if let quote = try source.lexQuote() {
+        members.append(.quote(quote))
+        continue
+      }
```

Comment:
@milseman Do we actually want it for the modern syntax `"..."` too? (this PR currently does both spellings)

Reply:
Sure, if it fall out naturally that way.